.expeditor/verify.pipeline.yml

```diff
@@ -500,6 +513,16 @@ steps:
       automatic:
         limit: 1
 
+  - label: "[unit] :windows: studio"
+    command:
+      - ./test/run_studio_test.ps1
+    agents:
+      queue: 'default-windows-privileged'
+    timeout_in_minutes: 20
+    retry:
```

Comment:
I think having  a single retry is beneficial some or most of the time.  If I know it takes 10+ minutes to complete a test suite, then I'm probably going to switch gears and respond to email, check slack, etc.  If a transient failure happens in the first few minutes then the auto retry will have saved me time over all.  The repo I have most experience with tends to fail fast or be more susceptible to transient issues, so that certainly influences my opinion, and it may not hold true to this code base.  It would certainly be interesting to see data on failure types and if retries are saving us time. 

That being said, I've reduced the timeouts to 5 minutes for these tests as they should never take more than a couple minutes. We can adjust upwards later if that proves to be too short.

Reply:
That sounds reasonable. The downside of retrying is directly proportional to the length of the timeout. The piece of data I think we lack (and which probably varies somewhat according to the particular step) is the prevalence of transient issues that have nothing to do with our code so the retry is a pure time-saver. If it's a transient issue in *our* code, I'd much rather have it fail and know about it.